backup/crowbar-backup

```diff
@@ -88,12 +88,8 @@ function check_resolv_conf
 {
   # check if DNS servers are available and can resolve domain names
   for ns in `grep "^\s*nameserver" /etc/resolv.conf | sed -e "s/\s*nameserver \([[:digit:]\.]\+\)[^[:digit:]\.]*/\1/" ` ; do
-    if ! netcat -zu $ns 53 ; then
-      echo "Error: Nameserver '$ns' is not available. Please reconfigure /etc/resolv.conf manually."
-      exit 1
-    fi
-    if ! getent hosts "$HOSTTEST" >/dev/null; then
-      echo "Error: Nameserver '$ns' can not resolve or ping $HOSTTEST. Please reconfigure /etc/resolv.conf manually"
+    if ! dig @$ns +short "$HOSTTEST" >/dev/null; then
```

Comment:
> Hrm, dig doesn't seem to return a non-0 value when there's no result.
> So -1.

@vuntz  it doesn't actually matter whether there's a result or not, it only matters if the server responds. if the server responds with "no such host", then we know the server is there. it properly returns a non-zero exit code if the server does not exist or is down.